### PR TITLE
Show a warning about Piskel/Jfxr/Yarn not being available yet for cloud projects on desktop

### PR DESCRIPTION
**Pull request by 4ian**

### CODE REVIEW

File: newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js [hunk 1]
```diff
@@ -17,6 +18,14 @@ const editors: Array<ResourceExternalEditor> = [
     editDisplayName: t`Edit with Piskel`,
     kind: 'image',
     edit: options => {
+      const storageProvider = options.getStorageProvider();
+      if (storageProvider.internalName !== 'LocalFile') {
+        Window.showMessageBox(
```
Comment (ClementPasteau) [inline newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js:23]: May I suggest you use the showAlert() for all those warnings.

File: newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js [hunk 2]
```diff
@@ -27,6 +36,14 @@ const editors: Array<ResourceExternalEditor> = [
     editDisplayName: t`Edit with Jfxr`,
     kind: 'audio',
     edit: options => {
+      const storageProvider = options.getStorageProvider();
+      if (storageProvider.internalName !== 'Jfxr') {
```
Comment (ClementPasteau) [inline newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js:40]: typo, "Jfxr" => "LocalFile"

**Review by ClementPasteau — APPROVED** [review]
typo & suggestion then ok

File: newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js (hunk 1 above)
Comment (4ian) [reply newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js:23]: Would unfortunately require to pass the function as parameter, and one usage is in SpritesList which is a class component, so can't use the context. Not taking the risk to refactor this now!

File: newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js (hunk 2 above)
Comment (4ian) [reply newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js:40]: Damn good catch

File: newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js (hunk 1 above)
Comment (4ian) [reply newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js:23]: (and ResourcesSelector too actually!)

**Pull request merged**